include/drivers/pcie/pcie.h

```diff
@@ -74,15 +81,14 @@ extern bool pcie_probe(pcie_bdf_t bdf, pcie_id_t id);
  * @brief Get the nth MMIO address assigned to an endpoint.
  * @param bdf the PCI(e) endpoint
  * @param index (0-based) index
- * @return the address, or PCIE_CONF_BAR_NONE if nonexistent.
+ * @return the address, size and flags of the mbar (address ist PCIE_CONF_BAR_NONE if nonexistent)
  *
  * A PCI(e) endpoint has 0 or more memory-mapped regions. This function
- * allows the caller to enumerate them by calling with index=0..n. If
- * PCIE_CONF_BAR_NONE is returned, there are no further regions. The indices
```

Comment:
I do not think this was correct. When I tested my changes on a Realtek 8111G Controller I got the following Bars
1) index=0 address=0xE000 size=256 type=IO
2) no mem bar at index 1
3) index=2 address=0x91404000 size=4k type=MEM (64 bit)
1) index=3 address=0x91400000  size=16k type=MEM (64 bit)

Reply:
bars that do not exist are now skipped, so this is true now